internal/pkg/manifest/validate_test.go

```diff
@@ -24,13 +24,14 @@ import (
 	"github.com/slackapi/slack-cli/internal/logger"
 	"github.com/slackapi/slack-cli/internal/shared"
 	"github.com/slackapi/slack-cli/internal/shared/types"
+	"github.com/slackapi/slack-cli/internal/slackcontext"
```

Comment:
🔍 Other `import` updates also remove `context` and I was wondering if we're using that package in other ways here?

Reply:
Edit: Ahhhh of course! It's in the setup of common mocks 👁️‍🗨️